app/controllers/dashboard-view-model.js

```diff
@@ -89,4 +90,37 @@ module.exports = class DashboardViewModel {
     // eslint-disable-next-line no-underscore-dangle
     return job.sourceUrl ? job.sourceUrl : i18n.__(`sourceType.${job.sourceType}`);
   }
+
+  dashboardTimeline(jobModels) {
+    const totals = { interested: 0, applied: 0, interview: 0, result: 0 };
+
+    jobModels.forEach(function (job) {
+      totals[job.status]++;
+    });
+
+    const totalsArray = Object.keys(totals).map(function (k) { return totals[k]; });
+    const maxTotal = Math.max.apply(Math, totalsArray);
```

Comment:
you can avoid the `Math.max.apply(Math, ...`) by using the ES6 [destructuring assignment](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)

```es6
const maxTotal = Math.max(...totalsArray);
```